views/components/collapsibleBlock.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * All About Olaf
+ * Individual row for inside a collapsible view
+ */
+
+import React from 'react'
+import {
+  View,
+  TouchableOpacity,
+} from 'react-native'
+
+import Collapsible from 'react-native-collapsible'
+
+
+export default class CollapsibleBlock extends React.Component {
+
+  static propTypes = {
+    borderColor: React.PropTypes.string.isRequired,
+    children: React.PropTypes.array.isRequired,
+  }
+
+  state = {
+    isCollapsed: true,
+  }
+
+  style() {
+    return {
+      container: {
+        marginBottom: 0,
+      },
+      collapsedContent: {
+        alignSelf: 'center',
+        backgroundColor: this.statusColor(),
+        
+      },
+      collapsedContainer: {
+      },
+    }
+  }
+
+  statusColor(){
```

Comment:
This function is redundant, it's clearer to just use `this.props.borderColor` wherever you need it.